### PR TITLE
Update WASM instructions

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

File: binding/angular/projects/rhino-web-angular/README.md [hunk 1]
```diff
@@ -157,4 +157,4 @@ Rhino requires a trigger to begin listening. To start listening for natural lang
 
 Custom contexts are generated using [Picovoice Console](https://picovoice.ai/console/). They are trained from text using transfer learning into bespoke Rhino context files with a `.rhn` extension. The target platform is WebAssembly (WASM), as that is what backs the Angular library.
 
-Convert the `.rhn` file to base64 and provide it as an argument to Rhino as in the above example. You may wish to store the base64 string in a separate JavaScript file and `export` it to keep your application code separate.
```
Comment (mrrostam) [inline binding/angular/projects/rhino-web-angular/README.md:160]: do we have it only for the angular binding? does it make sense to add it to all web-based bindings?

Comment (ErisMik) [reply binding/angular/projects/rhino-web-angular/README.md:160]: Yeah it seems like in Rhino we only have it for the Angular binding. I can add it to the other web-based bindings.

File: binding/react/README.md [hunk 2]
```diff
@@ -178,3 +178,9 @@ function VoiceWidget(props) {
     inferenceEventHandler
   );
 ```
+
```
Comment (mrrostam) [inline binding/react/README.md:181]: i know `vue` binding is not our best web binding, but still it can use some love here as well :)

Comment (ErisMik) [reply binding/react/README.md:181]: I knew I was missing one :)

**Review by mrrostam — APPROVED** [review]

**Pull request merged**